history.ts

```diff
@@ -0,0 +1,48 @@
+import { WeeklyArticleLog, WeeklyArticleHistory } from './types';
+import { get as getItem, set as setItem } from './lib/localStorage';
+import { getMondayFromDate } from './lib/dates';
+
+const historyWeeklyKey = 'gu.history.weeklyArticleCount';
+const mondayThisWeek = getMondayFromDate(new Date());
+
+/**
+ * Return the weekly article history object
+ */
+export const getWeeklyArticleHistory = (): WeeklyArticleHistory | undefined => {
```

Comment:
What are the tradeoffs between returning `undefined` vs an empty array here?

Reply:
`undefined` is better simply because the corresponding property in the Epic payload in DCR will be omitted from the JSON payload if the value is undefined - which then plays well on Contributions, which expects this as an optional prop (like the Epic view log). Either could work but `undefined` here will spare us from having to convert between that and empty arrays somewhere in the chain.